drivers/usb/device/usb_dc_sam0.c

```diff
@@ -119,9 +119,13 @@ static void usb_sam0_isr(void)
 	/* Acknowledge all interrupts */
 	regs->INTFLAG.reg = intflag;
 
+	/* End of reset interrupt */
 	if ((intflag & USB_DEVICE_INTFLAG_EORST) != 0U) {
 		UsbDeviceEndpoint *endpoint = &regs->DeviceEndpoint[0];
 
+		/* Enable suspend interrupt */
```

Comment:
nit: the comment doesn't add any new information.  Explain why this is happening or drop.

Also, as this starts adding state based interrupt selection, could you log a tracking bug and reference it from here as a TODO?